src/sentry/api/serializers/models/group.py

```diff
@@ -326,6 +327,7 @@ def serialize(
         self, obj: Group, attrs: MutableMapping[str, Any], user: Any, **kwargs: Any
     ) -> BaseGroupSerializerResponse:
         status_details, status_label = self._get_status(attrs, obj)
+        priority_label = self._get_priority(obj)
```

Comment:
```suggestion
        priority_label = obj.priority.name if obj.priority is not None else None
```

Reply:
Since priority is coming straight from the group model, it does not have a 'name' property

Reply:
Also, do we want to ever have the label as type 'None'?

Reply:
where is this getting used? if we're serializing for the FE, we can skip the field if the flag is off or the value is None.

Reply:
I'm adding this map in another PR - if it's useful feel free to copy it in and I'll deal with the merge conflict later lol:

```
PRIORITY_LEVEL_TO_STR: dict[int, str] = {
    PriorityLevel.LOW: "low",
    PriorityLevel.MEDIUM: "medium",
    PriorityLevel.HIGH: "high",
}
```

Reply:
@snigdhas On the GroupDetailsEndpoint(`GET /issues/<issue_id/`) when the data is serialized using GroupSerializerSnuba which has GroupSerializerBase as a parent.